packages/cli/templates/netlify-edge-functions/remix.config.js

```diff
@@ -0,0 +1,29 @@
+const { config } = require("@netlify/remix-edge-adapter");
+const baseConfig =
+  process.env.NODE_ENV === "production"
+    ? config
+    : {
+        ignoredRouteFiles: ["**/.*"],
+        serverModuleFormat: "cjs",
+        serverDependenciesToBundle: [
+          /@webstudio-is\//,
+          "nanoid",
+          "nanostores",
+          "@nanostores/react",
+          "@jsep-plugin/assignment",
+        ],
+      };
+
+/** @type {import('@remix-run/dev').AppConfig} */
+module.exports = {
+  ignoredRouteFiles: ["**/.*"],
+  ...baseConfig,
```

Comment:
edge workers needs all the dependencies to be bundled along with it.

Reply:
Have you tried 
```
  serverBuildPath: "build/index.js",
  serverConditions: ["worker"],
  serverMainFields: ["browser", "module", "main"],
  serverModuleFormat: "esm",
  serverPlatform: "neutral",
  serverDependenciesToBundle: "all",
  serverMinify: true,
```
I used above config to build for worker env

Reply:
The `baseConfig` that's coming from the `netlify-edge-adapter` has something for their own. Like they have `deno` in `serverCondition` and then in `serverBuildPath` it's `.netlify-edge-function/script.js` etc. And these are needed only for prod build and production serve according to their template.

Reply:
https://github.com/netlify/remix-compute/blob/main/packages/remix-edge-adapter/src/defaultRemixConfig.ts#L4